source/Core/InputValidator.php

```diff
@@ -174,37 +176,47 @@ public function checkEmail($user, $email)
      * Checking if user password is fine. In case of error
      * exception is thrown
      *
-     * @param User   $user                      Active user.
+     * @param string $email                     Email.
```

Comment:
As we can not change, what we pass to this method, argument should stay the same
```suggestion
     * @param User   $user                      Active user.
```